addons/survey/static/src/js/survey_form.js

```diff
@@ -46,6 +53,34 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
     // Handlers
     // -------------------------------------------------------------------------
 
+    _onKeyPress: function (event) {
+        var self = this;
+        var aIndex = 'a'.charCodeAt(0);
+        var zIndex = 'z'.charCodeAt(0);
```

Comment:
Is is really an "index" ?
zIndex is confusing because of css property....

Maybe zCharCode ?